src/wallet/tx_builder.rs

```diff
@@ -646,14 +649,22 @@ impl<'a, B, D: BatchDatabase> TxBuilder<'a, B, D, DefaultCoinSelectionAlgorithm,
 }
 
 /// Ordering of the transaction's inputs and outputs
-#[derive(Debug, Ord, PartialOrd, Eq, PartialEq, Hash, Clone, Copy)]
+#[derive(Clone)]
 pub enum TxOrdering {
     /// Randomized (default)
     Shuffle,
     /// Unchanged
     Untouched,
     /// BIP69 / Lexicographic
+    #[deprecated = "BIP69 does not improve privacy as was the intention of the BIP"]
```

Comment:
I don't know if we can, but if possible I would add a link to the discussion here on github. Maybe you could make a multiline warning, first line stays the same, and in a new line something like "To know more read the discussion at: ...".

Similar to what the rust compiler does when you have an error.

Reply:
@afilini Can you give me an example in the rust lang? I can just look at their source code and implement the same way.

Reply:
I meant something like:

```
warning: the feature `native_link_modifiers` is incomplete and may not be safe to use and/or cause compiler crashes
 --> library/unwind/src/lib.rs:8:12
  |
8 | #![feature(native_link_modifiers)]
  |            ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(incomplete_features)]` on by default
  = note: see issue #81490 <https://github.com/rust-lang/rust/issues/81490> for more information
```

I like that "note: see issue ..." at the end